client/core/core.go

```diff
@@ -2497,3 +2497,25 @@ func validateOrderResponse(dc *dexConnection, result *msgjson.OrderResult, ord o
 	}
 	return nil
 }
+
+// Logout logs the user out
+func (c *Core) Logout() error {
```

Comment:
This method is a bit out of place in the file.  It can go right after the exported `Login` method.